oidc_provider/views.py

```diff
@@ -253,12 +264,10 @@ def userinfo(request, *args, **kwargs):
         extra_claims = settings.get('OIDC_EXTRA_SCOPE_CLAIMS', import_str=True)(token)
         dic.update(extra_claims.create_response_dic())
 
-    response = JsonResponse(dic, status=200)
-    response['Access-Control-Allow-Origin'] = '*'
-    response['Cache-Control'] = 'no-store'
-    response['Pragma'] = 'no-cache'
+    success_response = JsonResponse(dic, status=200)
+    set_headers(success_response)
```

Comment:
Ah, now I see why `set_headers` was defined as a function, since it's used here too. How about moving it to global level (rather than being inside of the function) and renaming it to something more descriptive like `set_cache_and_cors_headers`?